tests/Functional/Net/SFTPTestCase.php

```diff
@@ -19,7 +19,7 @@
     public function setUp()
     {
         parent::setUp();
-        $this->scratchDir = uniqid('phpseclib-sftp-scratch-');
+        $this->scratchDir = uniqid('phpseclib-sftp-scratch-', true);
```

Comment:
@GrahamCampbell : this will be next fixer for CS Fixer when I finish with open PRs =)